client/src/plugins/version-info/ReleaseInfo.js

```diff
@@ -44,28 +44,35 @@ export function ReleaseInfo(props) {
+          We will now also use a more concise naming: <i>Camunda Platform</i> becomes Camunda 7
+          and <i>Camunda Cloud</i> becomes Camunda 8.<br />
+          Learn more about Camunda 8 in general <a href="toBeAdded">in the Camunda 8 blogpost.</a>
         </li>
         <li>
           <h4>Improved overall UI to become a "Visual IDE"</h4>
-          New welcome screen allows to easily create Platform and Cloud diagrams and forms.
-          The tab bar displays icons according to the file type and provides a new file menu via the plus button.
-          The properties panel has been completely redesigned. Tabs have been replaced with sections which can be opened and closed individually.
-          A granular breakdown with clear titles allows to identify relevant topics at a glance.
+          We re-worked the entire Modeler, including welcome screen, properties panel, status bar, tab bar, and bottom panel
+          following our visions of a "Visual IDE".<br />
+          Learn more about the changes <a href="https://camunda.com/blog/2022/01/camunda-modeler-5-0-0-alpha-0-released/">in this blog post</a>.
         </li>
         <li>
-          <h4>Support drilldown into collapsed subprocesses</h4>
-          With this release, we improved modeling of collapsed subprocesses for Camunda Platform.<br />
-          You can model collapsed subprocesses through the command pallet or by collapsing an expanded subprocess.
-          Use the blue drilldown icon to view and model the contents of the subprocess.
+          <h4>Process Debugging for Camunda 8 BPMN diagrams</h4>
+          The Modeler will now support you when automating BPMN processes with Camunda 8 by showing implementation errors,
+          and giving hints on how to fix them.
         </li>
         <li>
-          <h4>Process Debugging for BPMN</h4>
-          The Modeler will now validate and show if BPMN elements are not supported in the target platform.
+          <h4>Support drill down into collapsed subprocesses</h4>
+          In Camunda 7 BPMN diagrams, you can now model collapsed subprocesses by
+          collapsing an expanded subprocess. Use the blue drill down icon to view and model the contents of the subprocess.
+        </li>
+        <li>
+          <h4>New customization features for element templates</h4>
+          We added various new element templates features: <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/#groups">custom grouping</a>, <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/">documentation reference</a>, <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/">target element type</a>, and <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/#icons">custom task icons</a>.<br />
```

Comment:
The _target element type_ link does not have hash, so it links to the top of the docs.

Reply:
True, but the reason is that `targetElementType` is explained not in a seperate section but as part of the overall "What is a template" concept.